reports/hgdp_exploration/copy_hgdp_crams.py

```diff
@@ -0,0 +1,43 @@
+"""
+Copies HGDP CRAM files from 'gs://ibd-external-datasets'
+(update: gs://cpg-hgdp-broad-reprocessed-crams)
+to the fewgenomes main bucket.
+"""
+
+import os
+import csv
+import subprocess
+import hailtop.batch as hb
+
+# OUTPUT gets propagated from the analysis-runner cli to the server
+output_bucket = os.getenv('OUTPUT')
+assert output_bucket and output_bucket.startswith('gs://cpg-fewgenomes-temporary/')
+
+input_filelist = './data/filtered65.csv'
+analysis_runner_image = 'australia-southeast1-docker.pkg.dev/analysis-runner/images/driver:45c3f8125e300cd70bb790e32d96816f003a7af2-hail-0.2.64.devcb1c44c7b529'
+
+def copy_to_bucket(bucket: str, batch: hb.batch.Batch, sample_name: str, ftype: str, fname: str) -> None:
```

Comment:
This doesn't actually copy anything, so the name is a bit misleading. Maybe rename and add documentation on what this function does?

Reply:
As in, it declares a Batch job that copies the given file into the given bucket?